cmd/fs/cp.go

```diff
@@ -190,6 +203,23 @@ func newCpCommand() *cobra.Command {
 			return c.cpDirToDir(sourcePath, targetPath)
 		}
 
+		// Check if the target path has a trailing directory separator, indicating it should be a directory.
+		// If it has a trailing separator, verify the directory exists
+		normalizedTargetPathForCheck := filepath.ToSlash(targetPath)
+		if hasTrailingDirSeparator(fullTargetPath) || hasTrailingDirSeparator(normalizedTargetPathForCheck) {
```

Comment:
I think you can just trim targetPath here and let the rest of the code go into case 2 below